production.rb

```diff
@@ -90,7 +90,7 @@
 
   config.action_mailer.smtp_settings = {
     address: ENV['SMTP_HOST'],
-    port:    ENV['SMTP_PORT'],
+    port:    ENV['SMTP_PORT'].to_i,
```

Comment:
```suggestion
    port:    Integer(ENV.fetch('SMTP_PORT', '25'), 10),
```
I believe this is preferred over the `.to_i` method, although I don't remember where I got that from. This code will use 25 as a default value if the `SMTP_PORT` isn't set in the environment, so that the conversion to integer doesn't fail.